backend/tests/routes/Q&A/answers/approve.test.js

```diff
@@ -0,0 +1,49 @@
+const chai = require('chai');
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const server = require('../../../../index');
+
+chai.use(chaiHttp);
+
+describe('Approve test for Answers', () => {
+  let qId = '';
```

Comment:
[question] why do we need `aId` here?

Reply:
To update`isApproved` , we will need `id` of the answer.
So `qId` is for fetching the answer array and `aId` is for updating `isApproved` of that answer

Reply:
Why do we need `qId` for reaching answer even after we have `aId`? Am I missing here anything?

Reply:
Because we used `qId` to fetch answers to get the `aId`.

I have worked on Tests for first time. So there could be mistakes, you should check the code.
There could be possible optimization in it.

Reply:
Ah, I can see you wrote test for all the apis related to Q&A. Makes sense. Thanks for the clarification